src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -784,14 +834,16 @@ def gtdb_tk(root_dir, kbase_collection, load_ver):
     result_dir = _locate_dir(root_dir, kbase_collection, load_ver, tool='gtdb_tk')
     batch_dirs = _get_batch_dirs(result_dir)
 
-    summary_files = ['gtdbtk.ar53.summary.tsv', 'gtdbtk.bac120.summary.tsv']
-    genome_id_col = 'user_genome'
+    genome_id_col = loader_common_names.GENOME_ID_COL
     for batch_dir in batch_dirs:
-        summary_file_exists = False
 
+        summary_files = [file_name for file_name in os.listdir(os.path.join(result_dir, batch_dir)) if 
+                         re.search(loader_common_names.GTDB_SUMMARY_FILE_PATTERN, file_name)]
+        
+        summary_file_exists = False
```

Comment:
You already know whether there are any summary files via `len(summary_files)`

Reply:
https://github.com/kbase/collections/pull/329#discussion_r1231634713

Reply:
:+1: